README.rst

```diff
@@ -92,6 +92,58 @@ Icons, pictures, etc. used by the theme MUST be in a subdirectory named the
 same as the theme (minus the ".css") in the same directory as the theme so
 update the paths accordingly. See the included theme "default.css".
 
+
+UI Settings
+-----------
+
+Table columns override
+~~~~~~~~~~~~~~~~~~~~~~
+Choose which columns are visible in the incidents table.
+
+Via JSON env
+^^^^^^^^^^^^
+
+`table_fields` is a list of dictionaries, each representing a table column that will be displayed.
+Format::
+
+    {
+        "name": str,  # identifier
+        "label": str,  # display value
+        "cell_template": str  # path to template
+
+        "context": Optional[dict]  # context for the template
+    }
+
+Export an environment variable to configure columns that are displayed::
+
+    ARGUS_UI='{"table_fields": [{"name": "id", "label": "id2", "cell_template": "htmx/incidents/_incident_pk.html"}, ...]}'
```

Comment:
.. I do

```
export ARGUS_EXTRA_APPS=`cat extra.json`
```

in the shellscript I use to wrap manage.py. "extra.json" is primly indented for maximum human editability =)